udata/harvest/backends/ods.py

```diff
@@ -22,6 +22,7 @@ class OdsHarvester(BaseBackend):
     LICENSES = {
         "Open Database License (ODbL)": "odc-odbl",
         "Licence Ouverte (Etalab)": "fr-lo",
+        "Licence ouverte / Open Licence": "fr-lo",
```

Comment:
`Ouverte`? I don't know the official naming but on of these two lines needs to be adapted.